modules/cli/src/main/scala/coursier/cli/install/Install.scala

```diff
@@ -45,24 +45,10 @@ object Install extends CaseApp[InstallOptions] {
       .withNativeImageJavaHome(Some(graalvmHome))
 
     if (params.installChannels.nonEmpty) {
+      val progName = coursier.cli.Coursier.progName
+      val options = params.installChannels.flatMap(c => Seq("--add", c)).mkString(" ")
+      System.err.println(s"Warning: the --add-channel option is deprecated. Use '$progName channel $options' instead.")
 
-      // TODO Move to install module
```

Comment:
Let's keep the code that was removed too, so that a deprecation warning is printed, but the channels are actually added. The goal would be to nudge people to migrate to the new command, but not disrupt anyone's scripts using it.

Reply:
Or rather, call the equivalent logic in `coursier.cli.channel.ChannelCommand` from here.

Reply:
makes sense. Called the logic to add channel from here